### ISSUE TITLE
Add germline variant calling test data

### ISSUE THREAD

**Issue by drpatelh** [seqeralabs/nf-sentieon#2]
### Description of feature

We are currently using a minimal test dataset for SARS-CoV-2 which is sufficient to test the pipeline but we don't have dbSNP and indel files for this reference.

It would be good to have an additional `-profile test_germline` for test data created by Sentieon as part of their [Quick start docs](https://support.sentieon.com/quick_start/index.html#run-the-quick-start-package). This is a small dataset for germline variant calling from part of NA12878/HG001.

**Comment by drpatelh** [comment]
It would be nice to host the uncompressed data on S3 somewhere so we can use links to individual reference/input files. One option would have been to upload them to Github like we do on nf-core but the FastQ files are too large (~100M).

**Comment by DonFreed** [comment]
Agreed that the Sentieon Quickstart package is too large for this case.

Maybe we can leverage test datasets used for the Sarek (https://github.com/nf-core/test-datasets/tree/sarek) pipeline? In particular, the Sarek dataset contains trimmed dbSNP, Mills, and known indel VCFs along with small fastq files.
